lib/deploy/activate.rb

```diff
@@ -130,26 +132,18 @@ def default_logger
       logger
     end
 
-    def env_yaml_path
-      File.join(root, 'config/application_s3_env.yml')
-    end
-
     def root
-      File.expand_path('../../../', __FILE__)
+      Rails.root
```

Comment:
I'm pretty sure one of the reasons we didn't do `Rails.root` before is that this script is run outside of the Rails app, before Rails has loaded (or can load) and it helps set up the secrets that Rails needs --- I am not sure this will work